src/BuiltInTools/dotnet-format/CodeFormatter.cs

```diff
@@ -279,5 +279,31 @@ private static async Task<Solution> RunCodeFormattersAsync(
             formattableDocuments.AddRange(sourceGeneratedDocuments);
             return (projectFileCount + sourceGeneratedDocuments.Count, formattableDocuments.ToImmutable());
         }
+
+        private static bool IsFantomasInstalled()
+        {
+            var processStartInfo = new ProcessStartInfo
+            {
+                FileName = "dotnet",
+                Arguments = "tool list -g",
```

Comment:
IHMO Fantomas should be installed locally and not globally. 

This is allows people to configure per project the version of Fantomas they want to use. If we want to be exhaustive then we could check first for local version and then global version.

If none are installed, we can information the user of the tool possible ways to install Fantomas like they do on their documentation.

IHMO global tools are always a bad things because you have no control over the version installed.

Reply:
yes true, i generated this with copilot but i need way more iterations 🔁